.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "arrow-parens": "off",
+    "no-unused-vars": "off",
+    "no-plusplus":"off",
+    "import/extensions": "off"
```

Comment:
- [ ] This .estlint.config file is different from the one provided by Microverse. We should not change this file. We need to change to [this version](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) instead.